luigi/contrib/ftp.py

```diff
@@ -131,14 +131,11 @@ def _sftp_exists(self, path, mtime):
         return exists
 
     def _ftp_exists(self, path, mtime):
-        path_parts = path.split('/')
-        path = '/'.join(path_parts[:-1])
-        fn = path_parts[-1]
-
-        files = self.conn.nlst(path)
+        dirname = os.path.dirname(path)
```

Comment:
I did the split and join to separate the filename from the file path. `os.path.dirname(path)` is the better way to go about this.